### PR TITLE
Fix strong mode warnings

### PR DESCRIPTION
**Pull request by dramos07**

### CODE REVIEW

**Comment by dramos07** [comment]
I signed it!

File: lib/src/parameters.dart [hunk 1]
```diff
@@ -19,16 +19,18 @@ import 'exception.dart';
...
+
   /// The name of the method that this request called.
   final String method;
 
   /// The underlying value of the parameters object.
   ///
   /// If this is accessed for a [Parameter] that was not passed, the request
   /// will be automatically rejected. To avoid this, use [Parameter.valueOr].
-  final value;
+  get value => _value;
```
Comment (nex3) [inline lib/src/parameters.dart]: See https://github.com/dart-lang/vm_service_client/pull/8/files#r73964518

File: lib/src/parameters.dart [hunk 2]
```diff
@@ -302,12 +305,14 @@ class Parameter extends Parameters {
 
 /// A subclass of [Parameter] representing a missing parameter.
 class _MissingParameter extends Parameter {
+  @override
```
Comment (nex3) [inline lib/src/parameters.dart]: External packages generally don't use `@override`.

**Comment by nex3** [comment]
Done reviewing!

File: lib/src/parameters.dart (hunk 1 above)
Comment (dramos07) [inline lib/src/parameters.dart]: Sg

File: lib/src/parameters.dart (hunk 2 above)
Comment (dramos07) [inline lib/src/parameters.dart]: Sg

**Pull request closed without merging**